src/seedu/addressbook/parser/Parser.java

```diff
@@ -207,7 +207,7 @@ private Command prepareViewAll(String args) {
      * @throws ParseException if no region of the args string could be found for the index
      * @throws NumberFormatException the args string region is not a valid number
      */
-    private int parseArgsAsDisplayedIndex(String args) throws ParseException, NumberFormatException {
+    private static int parseArgsAsDisplayedIndex(String args) throws ParseException, NumberFormatException {
```

Comment:
There were more methods for you to change to static. 
Accordingly change how the method is called to a static method call.